input/cql/HIVEncounterElements.cql

```diff
@@ -84,168 +44,35 @@ define "Current Patient Age In Months":
 */
 // TODO: Replace placeholder with relevant CQL logic
 define "Gender":
```

Comment:
Probably the definitions for Gender / Female and Male are safe to migrate to HIVElements, but we should discuss.